Convenus/Api/Api.cs

```diff
@@ -130,6 +149,27 @@ private bool HasCurrentMeeting(List<CalendarEvent> events)
         {
             DateTime curTime = DateTime.Now;
             return events.Any(e => e.StartTime <= curTime && e.EndTime >= curTime);
+        }
+
+        private RoomStatus GetRoomStatus(List<CalendarEvent> events, int minutes)
+        {
+            if (events == null || events.Count == 0)

```

Comment:
thanks @jhofker, i think it's ok to use Count here though, since this is a list and the length of it is known, would be faster than calling the function.